### PR TITLE
update openldap readme

### PR DESCRIPTION
**Pull request by glicht**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
remove unused sections from openldap readme

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by glicht** [comment]
@anara123 @yaakovi added another step in circle to tigger content docs build if relevant README files have been updated upon merge to master.

File: Documentation/docs_site_update.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    # simply compare against origin/master. Local testing case..
+    DIFF_COMPARE=origin/master  # disable-secrets-detection
+elif [ "$CIRCLE_BRANCH" == "master" ]; then
+    # on master we use the range obtained from CIRCLE_COMPARE_URL
+    # example of comapre url: https://github.com/demisto/content/compare/62f0bd03be73...1451bf0f3c2a
+    # if CIRCLE_COMPARE_URL is not set we use last commit
+    if [ -z "$CIRCLE_COMPARE_URL" ]; then
+        DIFF_COMPARE="HEAD^1...HEAD"
+    else
+        DIFF_COMPARE=$(echo "$CIRCLE_COMPARE_URL" | sed 's:^.*/compare/::g')
```
Comment (yaakovi) [inline Documentation/docs_site_update.sh]: why not always use `HEAD^1...HEAD`?

File: Documentation/docs_site_update.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        if [ -z "${DIFF_COMPARE}" ]; then
+            echo "Failed: extracting diff compare from CIRCLE_COMPARE_URL: ${CIRCLE_COMPARE_URL}. Fail.."
+            exit 1
+        fi
+    fi
+else
+    DIFF_COMPARE=origin/master...${CIRCLE_BRANCH}
+fi
+
+DIFF_RES=$(git diff --name-only  "$DIFF_COMPARE"  | grep -E '(Integrations|Scripts|Playbooks)/.*README.md' )
```
Comment (yaakovi) [inline Documentation/docs_site_update.sh]: should this be modified in the future for packs READMEs? if so, better to add a TODO comment here

File: Documentation/docs_site_update.sh (hunk 1 above)
Comment (glicht) [reply Documentation/docs_site_update.sh]: It depends on how the merge was done and the build triggered. The build can have multiple commits so HEAD^1 is not accurate. So compare url is preferred. There are also some edge scenarios that circle compare url is empty.

File: Documentation/docs_site_update.sh (hunk 2 above)
Comment (glicht) [reply Documentation/docs_site_update.sh]: Still don't know if and what we will do regarding readme of packs. When we get there we can revist.

**Pull request merged**